testsuite/unittests/test_iaf_ps_dc_accuracy.sli

```diff
@@ -340,8 +340,6 @@ V <- ( mV.) <- endl
 endl
 
 (               h in ms    ) <-
-(        alpha_canon [mV]) <-
```

Comment:
Why do you remove these lines?

Reply:
`iaf_alpha_canon` was previously removed from the test as it is deprecated. I noticed the output was still here in the test when working on this PR because the test failed. I can add it again and remove it in another PR?

Reply:
No, fixing it here is fine.